README.md

```diff
@@ -20,7 +20,58 @@ spago install machines
 
 ## Quick start
 
-The quick start hasn't been written yet. Contributions are welcome!
+The examples here use `Identity` for simplicity. Usually, you would use
+a different `Monad`, such as `Effect`, `Aff`, `State`, etc.
```

Comment:
I think it would be nice to have a one-line description of the `MealyT` type before mentioning `Identity`. For example:

```suggestion
Mealy machines are finite state machines. The `MealyT f s a` type represents a machine where `f` is the effect used for evaluation, `s` is the input state, and `a` is the output value. The examples here use `Identity` as the effect type for simplicity, but you would usually use a different `Monad` such as `Effect`, `Aff`, or `State`.
```

Reply:
I'm not sure if I'm right but I think that `s` should be rather called an `input`. If my understanding of `MealyT` type is correct I'm opting for changing parameters names to `MealyT f i o` too.
P.S.
A somewhat interesting topic for a broader discussion from my perspective is: how this type really differs from the `Star f i o` from profunctors... ;-)

Reply:
`i` and `o` make more sense to me as well, but I'm not familiar with why `s` was originally chosen. I am personally fine with changing it and just referring to a machine's input and output throughout documentation comments and the quick start, but I don't have a strong opinion on it.

Reply:
I think it might be nicer to rename these to `input` and `output`, but I would do it in a subsequent PR because it would be out of place to change the module and module docs in this PR.

Reply:
👍 sounds good to me.